lib/services/livesync/platform-livesync-service.ts

```diff
@@ -39,6 +43,10 @@ export abstract class PlatformLiveSyncServiceBase implements IPlatformLiveSyncSe
 			let canExecute = this.getCanExecuteAction(platform, appIdentifier);
 			let action = (device: Mobile.IDevice): IFuture<void> => {
 				return (() => {
+					const normalizePlatformName = this.$mobileHelper.normalizePlatformName(platform);
+					const deviceType = device.isEmulator ? "emulator" : "device";
```

Comment:
from the Device do we get to know the particular OS of the device? Do we want to track this?